icons/googlecampaignmanager360.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Google Campaign Manager 360</title><path d="M15.8227 17.112c.1468 0 .2911-.0143.4367-.0213-.2558 1.6538-1.0194 3.2437-2.2943 4.518-2.5115 2.5104-6.2446 3.0395-9.2804 1.6036.9243-.0457 1.7896-.414 2.4488-1.0731.7074-.7074 1.0414-1.6197.9404-2.569-.041-.385-.1624-.77-.339-1.1433.8271.1366 1.6915-.127 2.29-.7255.5826-.5826.828-1.3835.7418-2.1623 1.477 1.0196 3.223 1.573 5.056 1.573zm7.3271-12.56c-.0458.8364-.3741 1.6605-1.0113 2.2985-.6654.6654-1.55 1.0319-2.491 1.0319-.434 0-.853-.0864-1.2471-.2357.1743.8568-.086 1.7651-.7107 2.3898-.9915.9916-2.6622 1.0652-3.7246.0028l-.0007.0006.0007.0007c-3.1955-3.1954-8.374-3.194-11.5695 0-3.1941 3.1942-3.1941 8.374 0 11.5682.581.581 1.3423.8715 2.1037.8715.7613 0 1.5227-.2905 2.1037-.8715a2.7317 2.7317 0 0 0 .3685-.4542c.0277-.043.0412-.0891.0657-.133.0646-.1156.1305-.2311.1732-.351.0325-.0913.0442-.1855.0645-.2787.0164-.0753.0434-.1494.052-.2256.0126-.1121.0045-.2255.0008-.3386-.002-.0587.0064-.117.0002-.1757-.0147-.1397-.0493-.2787-.0873-.4175-.0088-.032-.0103-.0645-.0203-.0964-.0509-.1633-.1194-.3246-.2005-.4832-.0035-.0068-.0051-.014-.0087-.0207-.171-.3288-.405-.644-.692-.9331-.0017-.0016-.0028-.0035-.0044-.005v-.0001c-.0018-.0018-.0031-.0038-.0049-.0056-.9356-.9356-1.1225-2.594-.0054-3.7124.9096-.9096 2.538-1.1688 3.7273-.0067l.0008.0006.5578.4926c3.2144 2.683 7.9996 2.5182 11.0178-.4987 2.5481-2.5471 3.0575-6.3544 1.5417-9.4138zm-9.934 3.8996c-.0769-.7803.191-1.5733.7548-2.137 1.046-1.046 2.7126-.9984 3.7164.0054a2.7594 2.7594 0 0 0 1.4329.7617 2.783 2.783 0 0 0 1.571-.1525 2.764 2.764 0 0 0 .9172-.6092c1.0829-1.0842 1.0829-2.8394 0-3.9224-3.1955-3.1954-8.374-3.194-11.5695 0-1.2737 1.2737-2.031 2.8642-2.289 4.5178.1438-.0069.2864-.021.4314-.021 1.824 0 3.5624.5472 5.0348 1.5572z"/></svg>
```

Comment:
I was still seeing some of the same differences as Adam was, @PratyushJayachandran, and your bottom left cut-out had gone a bit wibbly, so I gave it a try myself, if you want to run with it.
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Google Campaign Manager 360</title><path d="M15.8203 0c-2.093.0002-4.1858.7997-5.7832 2.3965-1.3364 1.3364-2.112 3.0208-2.33 4.7617a8.731 8.731 0 0 1 .4726-.0137c1.7849-.0001 3.5658.5466 5.0763 1.6387-.2066-.8822.0681-1.8261.7127-2.4707 1.046-1.0459 2.7113-.9977 3.7149.0059 1.0827 1.0826 2.8372 1.0826 3.92 0 1.0826-1.084 1.0826-2.8392 0-3.922C20.006.7992 17.9133-.0001 15.8202 0Zm7.0781 4.0781c.0785.929-.234 1.8855-.9414 2.5938-.9808.9809-2.438 1.2037-3.6386.6738.3115.9383.065 1.9896-.6329 2.6875-.9913.9913-2.6633 1.0662-3.7255.0039-.0173-.0172-.0293-.0352-.046-.0527l-.003.002c-3.1988-3.142-8.3393-3.1251-11.5165.0507-3.1935 3.1935-3.1935 8.373 0 11.5664 1.1618 1.1618 3.0453 1.1618 4.207 0 1.1618-1.1618.8689-2.7562-.293-3.918-.9353-.9354-1.1225-2.5928-.0058-3.7109.9095-.9095 2.5376-1.1696 3.7266-.0078.0726.0726.1477.1405.2246.2031 3.208 2.9849 8.2258 2.9155 11.3496-.207 2.68-2.679 3.1113-6.7548 1.295-9.8848zM10.7246 15.1992c.2151.8774-.0348 1.8317-.7012 2.498-.6718.672-1.6786.924-2.5918.6583.2105.3921.3471.8085.3926 1.2363.088.8272-.1925 1.6886-.8691 2.3652-.7565.7566-1.781 1.0896-2.7735 1 3.1157 1.7478 7.1304 1.2962 9.7813-1.3535 1.337-1.3364 2.1102-3.0208 2.3281-4.7617-1.946.1045-3.9194-.4446-5.5664-1.6426z"/></svg>
```